FpgaConfigBuilder/src/builder/DestinationConfigDirector.java

```diff
@@ -0,0 +1,22 @@
+package builder;
+
+import java.net.UnknownHostException;
+
+import datastructures.FpgaConfig;
+
+//e.g. enableMapping (output map part)
+public class DestinationConfigDirector { 
```

Comment:
Knows the steps to construct destination config object.

Similar class has to be developed for source config.